mailbox/api/src/test/java/org/apache/james/mailbox/util/EventCollector.java

```diff
@@ -22,11 +22,11 @@
 import java.util.Collection;
 import java.util.concurrent.ConcurrentLinkedDeque;
 
-import org.apache.james.mailbox.events.Event;
-import org.apache.james.mailbox.events.Group;
-import org.apache.james.mailbox.events.MailboxListener;
+import org.apache.james.events.Event;
+import org.apache.james.events.EventListener;
+import org.apache.james.events.Group;
 
-public class EventCollector implements MailboxListener.GroupMailboxListener {
+public class EventCollector implements EventListener.GroupEventListener {
```

Comment:
Didn't you recreate this class in events module? it's a duplicate then and should be maybe removed here?

Reply:
> Didn't you recreate this class in events module? 

Yes

> it's a duplicate then and should be maybe removed here?

No. I wanted to avoid a painful testing dependency.